dex/order/order.go

```diff
@@ -124,9 +130,6 @@ type Order interface {
 	// Order Management' section of the DEX specification.
 	Serialize() []byte
 
-	// SerializeSize gives the length of the serialized order in bytes.
-	SerializeSize() int
```

Comment:
We weren't using this outside of the order package, so I've unexported it for now.